kubernetes-deployment/bootstrap.py

```diff
@@ -407,9 +433,27 @@ def option_validation(args):
         if args.deploy or args.clean:
             print "You could not specify the option (-a or -r) with the option (-d or -c)"
             return False
+        if args.maintain != None:
+            print "You could not specify the option (-a or -r) with the option -m"
         # Add or remove node-list
         return True
 
+    if args.maintain != None:
+
```

Comment:
If add, remove and maintain are mutual exclusives, suggest put them as the values of a common parameter. For example, replace "-a", "-r", "-m" with a single acton command "-a" with values "add", "remove", "repair".

So internally you can use switch statement to handle them easily.

And internally, you can make "repair" and all the other maintenance command goes into a same branch like:

if args.action="repair" or args.action="xxx".

So generally you could make the mutual exclusive commands to be values of one parameter. This will makes users to do single choice naturally.